📝 Rework local tethered development (#2933)
platformsh/platformsh-docs: docs/src/guides/django/local/tethered.md

## Create the tethered connection

{{% tethered-dev/steps-start %}}

1.  To ensure your `settings.py` file acts as if in a Platform.sh environment,
    mock two variables present in active environments:

    ```bash
    export PLATFORM_APPLICATION_NAME=django && export PLATFORM_ENVIRONMENT=new-feature
    ```

> 1.  Install dependencies by running the command for your package manager:

Reviewer: ```suggestion
1.  To install dependencies, run the command for your package manager:
```

    {{< /codetabs >}}

{{% tethered-dev/steps-end %}}

{{% local-dev/next-steps-start %}}

>     Fill it with the following example, depending on your package manger:

Reviewer: 📋 
```suggestion
    Fill it with the following example, depending on your package manager:
```
